base/intset.jl

```diff
@@ -143,7 +143,7 @@ function intersect!(s1::IntSet, s2::IntSet)
 end
 
 setdiff(s::IntSet, ns) = setdiff!(copy(s), ns)
-setdiff!(s::IntSet, ns) = (for n in ns; _delete!(s, n); end; s)
+setdiff!(s::IntSet, ns) = (for n in ns; delete!(s, n); end; s)
```

Comment:
At least the optimized method (below) when `ns` is an `IntSet` remains unaffected. It would also be easy to have an `unsafe_setdiff!` if needed (till now, it seems `@boundscheck` is not used for this kind of usage?)